src/features/get_rating/interface.rs

```diff
@@ -1,7 +1,8 @@
+//! Contains trait implementations for [`AppService`] and other app definitions.
 use crate::app::AppContext;
 
 use crate::features::{
-    app::{service::AppService, use_cases},
+    get_rating::{service::AppService, use_cases},
```

Comment:
It feels off having the service still referred to as app, but it will be a breaking change for the App Center to change the proto names, so probably better to leave it for now.

I like call it rating over app, matches closer to what is actually being dealt with in this module. My only comment is it feels inconsistent to have the verb "get" in front of the feature name?

Reply:
I guess, that's true, and it's more rusty to just call it "rating", my thinking was you can also set a rating but that's closer to `cast_vote` than `set_rating`.